docs/developer-docs/integrations/sns/launching/launch-summary.md

```diff
@@ -0,0 +1,332 @@
+| Treasury account on the SNS Ledger                | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+| Swap account on the SNS Ledger                    | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+
+
+
+- #### Step 8: Anyone in the community submits an NNS proposal to start the decentralization swap.{#SNS-launch-step-startSwap}
+  
+  Once the control of the dapp has been handed over to the SNS, anyone in the community can send an NNS proposal to trigger the decentralization swap. This proposal defines the conditions for the decentralization swap (e.g. how many ICP tokens should at least and at most be collected).
+
+If successful, at the end of stage, we have:
+
+|  Decentralization swap state                      | PENDING                                    |  
+|-------------------                                | ----------------                        |
+
+
+|  Objects in an app subnet                                      | State                    |  Controlled by |
+|-------------------                                | ----------------                      | ----------------|
+| A dapp                                            | Operational                           | ~~Dapp developer principal~~ Newly-created SNS canisters | 
+
+|  Objects in the SNS subnet                                       | State                  |  Controlled by |
+|-------------------                                | ----------------                      | ----------------|
+| ~~A principal that can deploy to SNS subnet~~     | Revoked because it is 1-time use      | NA                        |
+| Newly-created SNS canisters on the SNS subnet     | Pre-decentralization swap mode        | Initial developer neurons |  
+| Initial developer neurons                         | Pre-decentralization swap mode        | Dapp developer principal  |
+| Treasury account on the SNS Ledger                | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+| Swap account on the SNS Ledger                    | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+
+- #### Step 9: NNS DAO accepts or rejects the proposal
+
+  This is the **last of three** proposals that need to successfully pass for the process to continue. 
+
+  When voting on this proposal, the NNS voters can verify the parameters in the already
+  existing SNS canisters as well as the swap parameters that are set in the proposal.
+   
+  * If the NNS proposal is adopted, the swap is started after a specified delay. 
+  * If the NNS proposal is rejected, the SNS launch is aborted and the dapp’s control is handed
+  back to the original developers of the dapp.
+
+  :::info
+  Once this NNS proposal passes, the rest of the process should complete.
```

Comment:
Even though the process completes, completion could also mean that the swap fails and control is given back to the developers. Should it maybe be explained what we mean by "complete"?

My interpretation is actually that in each case "the process completes" it might just be taht at completion the SNS canisters are not useful. But in all cases, either the dapp is under a functional SNSs' control or the dapp is again in the control of the devs. 
Maybe the even more relevant information here would be "even if this adopted, the swap may still fail" ?